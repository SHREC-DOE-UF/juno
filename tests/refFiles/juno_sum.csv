ComponentName, StatisticName, StatisticSubId, StatisticType, SimTime, Rank, Sum.u64, SumSQ.u64, Count.u64, Min.u64, Max.u64
cpu:customhandler, calls-to-rand, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
cpu:customhandler, calls-to-rseed, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
cpu, cycles, , Accumulator, 125119182, 0, 300046, 300046, 300046, 1, 1
cpu, instructions, , Accumulator, 125119182, 0, 300004, 300004, 300004, 1, 1
cpu, mem-reads, , Accumulator, 125119182, 0, 2, 2, 2, 1, 1
cpu, mem-writes, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
cpu, add-ins-count, , Accumulator, 125119182, 0, 100000, 100000, 100000, 1, 1
cpu, sub-ins-count, , Accumulator, 125119182, 0, 100000, 100000, 100000, 1, 1
cpu, div-ins-count, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
cpu, mod-ins-count, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
cpu, mul-ins-count, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
cpu, and-ins-count, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
cpu, or-ins-count, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
cpu, xor-ins-count, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
cpu, not-ins-count, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetch_opportunities, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetches_issued, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetches_canceled_by_page_boundary, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetches_canceled_by_history, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, Prefetch_requests, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, Prefetch_drops, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetS_I, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
l1cache, stateEvent_GetS_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetS_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetX_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetX_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetX_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetSX_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetSX_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetSX_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetSResp_IS, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetXResp_IS, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
l1cache, stateEvent_GetXResp_IM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetXResp_SM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Inv_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Inv_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Inv_IS, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Inv_IM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Inv_SM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Inv_SB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Inv_IB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_IS, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_IM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_SB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_IB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Fetch_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Fetch_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Fetch_IS, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Fetch_IM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Fetch_SM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Fetch_IB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_Fetch_SB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_IS, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_IM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_SM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_SB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_IB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_IS, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_IM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_SM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_SB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_IB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLineResp_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLineResp_IB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLineResp_SB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_GetS, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
l1cache, eventSent_GetX, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_GetSX, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_PutM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_NACK, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_FlushLine, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_FlushLineInv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_FetchResp, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_FetchXResp, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_AckInv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_GetSResp, , Accumulator, 125119182, 0, 2, 2, 2, 1, 1
l1cache, eventSent_GetXResp, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_FlushLineResp, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_Put, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_Get, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_AckMove, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_CustomReq, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_CustomResp, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_CustomAck, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, EventStalledForLockedCacheline, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, evict_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, evict_S, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, evict_M, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, evict_IS, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, evict_IM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, evict_SM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, evict_SB, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_GetS_hit, , Accumulator, 125119182, 0, 3, 9, 1, 3, 3
l1cache, latency_GetS_miss, , Accumulator, 125119182, 0, 31, 961, 1, 31, 31
l1cache, latency_GetX_hit, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_GetX_miss, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_GetX_upgrade, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_GetSX_hit, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_GetSX_miss, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_GetSX_upgrade, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_FlushLine, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_FlushLine_fail, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_FlushLineInv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, latency_FlushLineInv_fail, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSHit_Arrival, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
l1cache, GetXHit_Arrival, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSXHit_Arrival, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSHit_Blocked, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetXHit_Blocked, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSXHit_Blocked, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSMiss_Arrival, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
l1cache, GetXMiss_Arrival, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSXMiss_Arrival, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSMiss_Blocked, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetXMiss_Blocked, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSXMiss_Blocked, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_AckPut_I, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_PutS, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, eventSent_PutE, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetch_evict, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetch_inv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetch_useful, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetch_coherence_miss, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, prefetch_redundant, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetS_E, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
l1cache, stateEvent_GetX_E, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_GetSX_E, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_E, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_E, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_E, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_ForceInv_E, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_E, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, evict_E, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, TotalEventsReceived, , Accumulator, 125119182, 0, 3, 3, 3, 1, 1
l1cache, TotalEventsReplayed, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetS_uncache_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetX_uncache_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSX_uncache_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSResp_uncache_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetXResp_uncache_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, CustomReq_uncache_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, CustomResp_uncache_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, CustomAck_uncache_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, NULLCMD_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetS_recv, , Accumulator, 125119182, 0, 2, 2, 2, 1, 1
l1cache, GetX_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSX_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, FlushLine_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, FlushLineInv_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetSResp_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, GetXResp_recv, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
l1cache, FlushLineResp_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, Inv_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, ForceInv_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, Fetch_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, FetchInv_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, FetchInvX_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, NACK_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, AckPut_recv, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
l1cache, MSHR_occupancy, , Accumulator, 125119182, 0, 28, 28, 300046, 0, 1
l1cache, Bank_conflicts, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
memory, requests_received_GetS, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
memory, requests_received_GetSX, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
memory, requests_received_GetX, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
memory, requests_received_PutM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
memory, outstanding_requests, , Accumulator, 125119182, 0, 11, 11, 125119, 0, 1
memory, latency_GetS, , Accumulator, 125119182, 0, 11, 121, 1, 11, 11
memory, latency_GetSX, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
memory, latency_GetX, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
memory, latency_PutM, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
memory, cycles_with_issue, , Accumulator, 125119182, 0, 1, 1, 1, 1, 1
memory, cycles_attempted_issue_but_rejected, , Accumulator, 125119182, 0, 0, 0, 0, 0, 0
memory, total_cycles, , Accumulator, 125119182, 0, 125119, 15654764161, 1, 125119, 125119
