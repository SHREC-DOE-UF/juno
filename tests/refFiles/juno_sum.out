Juno[cpu:JunoCPU:0]: Loading memory interface: memHierarchy.memInterface ...
Juno[cpu:JunoCPU:0]: Successfully loaded memory interface.
Juno[cpu:JunoCPU:0]: Opening program ../asm/sum.bin ...
Juno[cpu:JunoProgramReader:0]: Application Version:      1000000
Juno[cpu:JunoProgramReader:0]: Data Length:              16
Juno[cpu:JunoProgramReader:0]: Inst Length:              28
Juno[cpu:JunoProgramReader:0]: Padding                   4
Juno[cpu:JunoCPU:0]: Creating an instruction manager...
Juno[cpu:JunoCPU:0]: PC set to: 16
Juno[cpu:JunoCPU:0]: Creating a register file of 16 64-bit integer registers...
Juno[cpu:JunoCPU:0]: Creating load/store unit...
Juno[cpu:JunoCPU:0]: Loading custom instructions...
Juno[cpu:JunoCPU:0]: Loaded 1 custom instruction handlers.
Juno[cpu:JunoCPU:0]: Loading operation cycle counts...
Juno[cpu:JunoCPU:0]: Configuring statistics...
Juno[cpu:JunoCPU:0]: Initialization done.
Juno[cpu:init:0]: Sending initialization data to memory...
Juno[cpu:init:0]: Initialization data sent.
Simulation is complete, simulated time: 125.119 us
